Feature/51 papermill parameters

rknuu opened the pull request:
1. Implemented papermill's parameterize feature (created a cell near the beginning that can be parameterized as detailed [in their docs](https://papermill.readthedocs.io/en/latest/usage-parameterize.html)
2. Updated Slurm scripts and notebook such that it parameterizes `TFTTransformerepochs`, pulled other configurables up to this cell (only one cell can have the values substituted) and to handle branches / forks of the repo. 
3. Implemented all slurm scripts so that a pattern can be built for cloudmesh-sbatch
4. Switched the execution logic in the slurm scripts to be based on a timestamp, so we can have concurrent executions on the same cards should we need to.

An example of how this works:
```bash
# Below are the defaults and don't need to be specified, but can be modified to change how the program runs
FORK="laszewsk"
BRANCH="main"
RUNSTAMP="$(date)"
sbatch --export="BRANCH=${BRANCH},RUNSTAMP=${RUNSTAMP},FORK=${FORK}" rivanna-k80.slurm
```

Supports Data-ScienceHub/mlcommons-science#51

laszewsk commented:
would it be possible to have the slurm scripts checked in separately from the notebook? ITs difficult to say what it is without personal review

laszewsk commented:
Although we are using a papermill, we have not agreed that we use the papermill feature to populate the parameters or a yamlfile that is being read in.  We need to discuss this.

rknuu commented:
@laszewsk I have reverted and updated the notebook / slurm scripts to defer that for a later date (so we can review commit f7962165752f94c321d613ddb850dd11bd064770 at a later date if we give this a thumbs up).
